src/perf_counters.cc

```diff
@@ -38,7 +38,7 @@ PerfCounters PerfCounters::Create(
     return NoCounters();
   }
   if (counter_names.size() > PerfCounterValues::kMaxCounters) {
-    GetErrorLogInstance()
+    VLOG(2)
```

Comment:
i think this should still be an error as the user has attempted to set counters.

Reply:
The behavior is same even if the user doesn't give any benchmark_perf_counters command-line option.

Reply:
if the user doesn't set the flag the counter_names size is > maxCounters?

that seems like a bug further upstream no?

Reply:
Oops, that was for the comment below. I'll update this part.